File: WEEKLYCODE#2/plus_minus/C/plus_minus.c
#include <stdio.h>

int main(void) {
	int n,i,plus=0,minus=0,zero=0;
	scanf("%d",&n);
	int ar[n];
	//taking input
	for(i=0;i<n;i++)
	    scanf("%d",&ar[i]);
	//calculating number of elements >,<,= 0
	for(i=0;i<n;i++){
	    if(ar[i]>0)
	        plus++;
	    else if(ar[i]<0)
	        minus++;
	    else
	        zero++;
	}
	//type casting and printing float values
	printf("%f\n",(float)plus/n);
	printf("%f\n",(float)minus/n);
	printf("%f\n",(float)zero/n);
	return 0;
}

File: WEEKLYCODE#2/plus_minus/Java/plus_minus.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution
{
	public static void main(String[] args) {
	    Scanner input = new Scanner(System.in);
		int i,n,x,plus=0,minus=0,zero=0;
		n=input.nextInt();
		//taking input and calculating elements >,<,= 0
		for(i=0;i<n;i++){
		    x=input.nextInt();
		    if(x>0)
		        plus++;
		    else if(x<0)
		        minus++;
		    else
		        zero++;
		}
		//type casting and printing results
		System.out.println(plus/(float)n);
		System.out.println(minus/(float)n);
		System.out.println(zero/(float)n);
	}
}
